src/axom/primal/operators/evaluate_integral.hpp

```diff
@@ -40,23 +40,23 @@ namespace axom
 namespace primal
 {
 /*!
- * \brief Evaluate a line integral along a collection of Bezier curves
+ * \brief Evaluate a line integral along the boundary of a CurvedPolygon object 
+ *        on a scalar field.
  *
- * The line integral is evaluated on each curve in the array, and added
- * together to represent the total integral. The curves need not be connected.
+ * The line integral is evaluated on each curve in the CurvedPolygon, and added
+ * together to represent the total integral. The Polygon need not be connected.
  * Uses 1D Gaussian quadrature generated by MFEM.
- *
- * \param [in] cs the array of Bezier curve objects
- * \param [in] integrand the lambda function representing the integrand. 
- * Must accept a 2D point as input, and return a double (scalar field) or 
- * 2D vector object (vector field).
- * \param [in] npts the number of Gaussian quadrature nodes for each component
+ * 
+ * \param [in] cpoly the CurvedPolygon object
+ * \param [in] scalar_integrand the lambda function representing the integrand. 
+ * Must accept a Point<T, NDIM> as input and return a double
+ * \param [in] npts the number of quadrature points to evaluate the line integral
```

Comment:
Clarification: `npts` is the number of quadrature points per `BezierCurve` in the `CurvedPolygon`, right?